api/src/peoplemeasurement/queries.py

```diff
@@ -0,0 +1,97 @@
+def get_today_15min_aggregation_sql(datestr):
+    return f"""
+with RawData as(
+    select
+        sensor,
+        timestamp,
+        coalesce(avg((detail_elems ->> 'count')::integer) filter (where detail_elems ->> 'direction' = 'down'), 0) + coalesce(avg((detail_elems ->> 'count')::integer) filter (where detail_elems ->> 'direction' = 'up'), 0) as total_count,
+        coalesce(avg((detail_elems ->> 'count')::integer) filter (where detail_elems ->> 'direction' = 'down'), 0) as count_down,
+        coalesce(avg((detail_elems ->> 'count')::integer) filter (where detail_elems ->> 'direction' = 'up'), 0) as count_up,
+        avg((detail_elems ->> 'count')::numeric) filter (where detail_elems ->> 'direction' = 'density') as density_avg,
+        avg((detail_elems ->> 'count')::numeric) filter (where detail_elems ->> 'direction' = 'speed') as speed_avg
+    from peoplemeasurement_peoplemeasurement, jsonb_array_elements(details) detail_elems
+    where timestamp > '{datestr}'
```

Comment:
Deze query is op deze `where` na (en de `where total_count != 0` op regel 28) identiek aan de view die je maakt in de migration. 

Dit is niet ideaal, maar gezien de urgentie hiervan approve ik hem.